crates/typed-store/src/rocks/tests.rs

```diff
@@ -783,3 +699,35 @@ fn open_map<P: AsRef<Path>, K, V>(path: P, opt_cf: Option<&str>) -> DBMap<K, V>
 fn open_rocksdb<P: AsRef<Path>>(path: P, opt_cfs: &[&str]) -> Arc<RocksDB> {
     open_cf(path, None, MetricConf::default(), opt_cfs).expect("failed to open rocksdb")
 }
+
+#[tokio::test]
+async fn test_sampling() {
```

Comment:
Moved from crates/typed-store/tests/macro_tests.rs. All other tests in that test file are testing DBMapUtils macro